Выполнение задачи спринта 04

Shtrifel opened the pull request:
Задача четвертого спринта. Создание квиза из 10 вопросов.

Shtrifel committed:
Выполнение задачи спринта 04

tsaidov7 commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -2,9 +2,169 @@ import UIKit
 
 final class MovieQuizViewController: UIViewController {
```
![simulator_screenshot_D66DFE4E-E7C5-4E3A-8E0F-B660FFC9FE30](https://user-images.githubusercontent.com/37248555/209075506-8e4884a7-34e0-4173-9270-157844e82c35.png)

tsaidov7 replied:
Экран совершенно не соответствует макетам. Неверные шрифты, нет скруглений картинки, рамка после выбор ответа не исчезает, а остается на следующем вопросе, кнопки неверно расположены...

Shtrifel replied:
Шрифты - добавлены
Скругление - добавлено
Рамка после ответа не исчезает - исправлено
Кнопки - выровняны

tsaidov7 commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -2,9 +2,169 @@ import UIKit
+            let givenAnswer = true
+            showAnswerResult(isCorrect: givenAnswer == currentQuestion.correctAnswer)
+        
+    }
```
А почему код поехал? Отступы надо бы соблюдать для читаемости кода

Shtrifel replied:
Ок

tsaidov7 commented on MovieQuiz/Helpers/UIColor+Extensions.swift:
```diff
@@ -1,4 +1,11 @@
 import Foundation
 import UIKit
```
Не нужно импортить оба фреймворка. UIKit содержит в себе Foundation

Shtrifel replied:
Ок

Shtrifel committed:
Исправлены замечания:
1) Добавлены шрифты по макету
2) Добавлено скругление картинки и рамки
3) Исправлено отсутствие исчезания рамки с ответом при переходе на следующий вопрос
4) Выровнены кнопки
5) Убран лишний import
6) Выровнены строки кода
